docs/src/lib/binary_functions.md

```diff
@@ -101,6 +101,7 @@ pontryagin_difference
 ## Subset check
 
 ```@docs
+issubset
```

Comment:
i guess `issubset(::LazySet, ...)` so it doesn't print the methods from Base (`[1] issubset(a::BitSet, b::BitSet) in Base at bitset.jl:419`, ...)

Reply:
Did you try it? I didn't but I think `Documenter` only shows local module code because of this line:
https://github.com/JuliaReach/LazySets.jl/blob/8a7212b45032806529773007e46a91d4c1217710/docs/make.jl#L9

Reply:
I tried it and it works in Julia versions before v1.5. For v1.5 it does not work at all (does not find the associated function) but I could not find a quick solution, so I suggest that we fix that later in #2310.